src/tileLayer.js

```diff
@@ -1416,16 +1416,23 @@ module.exports = (function () {
       var map = this.map(),
           mapzoom = map.zoom(),
           roundzoom = this._options.tileRounding(mapzoom),
-          unit = map.unitsPerPixel(zoom === undefined ? roundzoom : zoom);
+          unit = map.unitsPerPixel(zoom === undefined ? roundzoom : zoom),
+          gcsPt;
       if (pt === undefined) {
         var size = map.size();
         pt = {x: size.width / 2, y: size.height / 2};
       }
+      /* displayToGcs can fail under certain projections.  If this happens,
```

Comment:
how can we replicate this behavior in our current version?

Reply:
just curious but agree on the statement

Reply:
In the reprojection example, choose a projection that curves at the edges, (ESRI:53009 or any Mo
llweide will work).  Zoom out.  If you zoomed out far enough, this code path is triggered.

Reply:
Got it, thanks.